tests/test_helper/lxd.bash

```diff
@@ -1,10 +1,9 @@
 function launch_containers() {
-    local image_name=$1; shift
     local containers=$*
 
     for container in $containers; do
         echo "# Launching $container" >&3
-        lxc launch -q "ubuntu:$image_name" "$container" < \
+        lxc launch -q "ubuntu:lts" "$container" < \
```

Comment:
The `lts` alias appears to be a reasonable default for the main branch, however we might want to lock down stable branches to a specific version.

What about retrieving the default from an environment variable, which defaults to `lts` when not set?

Reply:
Sure, env variable could be one way to set a default value. Thought, do we really have a use-case for configurable test images? The way I see it is that microovn, being snap, is heavily insulated from the underlying OS version so it makes very little difference what container version we use for testing. The only possible thing that could make a difference is a kernel version, but that can't be affected by specifying different container image. We'll still share the underlying host's kernel, that we can't influence.

My main intention with this change was to reduce the amount of stuff we need to think about, and change, whenever there's a new release.

Reply:
We do indeed get pretty good insulation from the environment being a confined snap, there are however other factors that come into play such as apparmor and difference in system defaults (found a [random bug](https://bugs.launchpad.net/snapd/+bug/1981546) as an example).

Not having to change things on new releases is a high priority, but we also don't want stable branches to break.

I'm fine with trying to have this rolling, but I'd really like to have the ability to control this via the environment from the start, so that we don't need to backport that change once a stable branch breaks.

Reply:
That is a very good point, and I was thinking about running regular (weekly?) CI runs for all branches, to discover these issue as they happen, not just when we have some backports. We could then run all stable branches with all available LTSes on regular bases.